src/resources/assets/useUserAsset.ts

```diff
@@ -22,11 +20,3 @@ export function useUserAsset(uniqueId: string) {
     }
   );
 }
-
-export function useUserNativeNetworkAsset(chainId: ChainId) {
```

Comment:
given my changes, we no longer needed this function since we're relying on Ben's maxSwappableAmount to handle the native gas calculations for us